core/modules/css/cssLib.i

```diff
@@ -47,7 +53,118 @@ namespace std {
     %template(StringVector) vector<string>;
 };
 
+%include "sql/SqlErrorObject.h"
+
+%{
+    void setPythonException(const std::string& exceptionStr, const lsst::qserv::css::CssError& ex) {
+        PyObject* module = PyImport_ImportModule("lsst.qserv.css");
+        if (!module)
+            throw;
```

Comment:
AFAIK it rethrows the current C++ exception (which does crash python). If the css module can't be imported I'm not really sure what other option there is... I guess it could raise a python exception, that would probably be better.